restcountries/base.py

```diff
@@ -167,9 +168,6 @@ def get_countries_by_capital(cls, capital, filters=None):
 
 
 class Country:
-    def __str__(self):
```

Comment:
Why remove that method?

Reply:
There are two __str__ methods defined for the same class, I removed the less verbose one that would be overshadowed by the second one anyway.